.github/workflows/docker_build.yaml

```
@@ -22,6 +23,16 @@ jobs:
             latest
```

```suggestion
            ${{ github.event.client_payload.setLatest && 'latest'}}
```
*possible doesnt work like this